content/apps/catalogs.md

```diff
@@ -104,12 +103,64 @@ At any moment, you can verify if a catalog is enabled by calling the [get catalo
 
 ### Step 3: Get products using catalogs
 
+Once you have an enabled catalog, you can paginate the corresponding product UUIDs using this [endpoint](/api-reference.html#Appcatalogproduct).
```

Comment:
```suggestion
Once you have an enabled catalog, you can paginate the corresponding products using this [endpoint](/api-reference.html#get_app_catalog_products).
```

Why don't you recommend the "Get the list of products related to a catalog" instead of "Get the list of product uuids", it's probably more useful for them.

Reply:
Because this documentation was written before having the products endpoints.